templates/todo/projects/python-mongo-aca/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Some of these devcontainer.json files are the same.  We can re-arrange how we store them in common and then copy any that are the same out to the various templates.

Reply:
Fixed in my commit